app-misc/vcontrold/vcontrold-0.98.11.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2020 Gentoo Authors
```

Comment:
You should use our automated tools to update these ebuilds, in this case a tool called **pkgdev** so it updates the copyright year automatically. It also adds a better commit summary for you to fill.